challenge_templates/pwn/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:latest
```

Comment:
I know it's just a template but should this be fixed to a version?  Eg soon `latest` will change from `16.04` to `18.04` and might break some pwns if it is rebuild

Reply:
Agreed, does setting it to `ubuntu:xenial` okay?

Reply:
Yep that should do it